### PR TITLE
Create olkrm.json

### LINKED ISSUE #1598: Adding Pittsburgh
## Adding <your-city> to the map

Hi everyone! This issue is to track the addition of <your-city> to the map, which can be found at:

- Longitude: 40.426121
- Latitude: -80.016208

### PR DESCRIPTION
**Pull request by Olkrm**
the inclusion of Pittsburgh, PA.
closes #1598

### CODE REVIEW

**Comment by JoshuaMJ2001** [comment]
Your mum

**Comment by JoshuaMJ2001** [comment]
:wink:

**Comment by Briggsy1213** [comment]
whay you evem do this bro, so bad :-1:

File: olkrm.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (crichID) [inline olkrm.json]: @Olkrm you will need to move this file into the _pins directory before you merge! Simply click on the **Files Changed** tab, then click on the pencil icon in the top right corner to edit. In the filename field, add `_pins/` to the beginning of the file name. Then scroll to the bottom and commit your changes! Once you have done that, I will approve your changes and you can merge!

**Pull request merged**